pkg/controller/certificatesigningrequest/ipsec_csr_approver.go

```diff
@@ -123,7 +132,8 @@ func (ic *ipsecCSRApprover) verifyCertificateRequest(req *x509.CertificateReques
 }
 
 func (ic *ipsecCSRApprover) verifyIdentity(nodeName string, csr *certificatesv1.CertificateSigningRequest) error {
-	if csr.Spec.Username != antreaAgentServiceAccountName {
+	c := newIPsecCSRApprover()
```

Comment:
you are creating a new instance of `ipsecCSRApprover, this is not right

Reply:
will revert and fix  : )